src/main/java/com/minecolonies/coremod/event/EventHandler.java

```diff
@@ -372,7 +373,7 @@ public void onLivingDeath(@NotNull LivingDeathEvent event)
     /*@SubscribeEvent
     public void onEntityJoinWorld(@NotNull EntityJoinWorldEvent event)
     {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)